### ISSUE TITLE
Windows support for some pillars of the rails ui

### ISSUE THREAD

**Issue by ktfth** [getrailsui/railsui#18]
I feel the lack of customization for windows support, some commands made the application crash and not give a pleasant developer experience in configuration for example. But with some effort the result, become more interestingly to use, after all the setup was made done on my try. If you have some guidance to contribute with these aspects, please share with me, I really like the project and gonna give more support for the tool soon.

Thank you for the awesome contribution with Rails UI.

**Comment by justalever** [comment]
Good call! I'll dig into this soon. I have to say I've only built, tested, and used this on MacOS so far but as the project grows I'll put in some effort here 👍
